src/main/resources/ath-container/Dockerfile

```diff
@@ -81,7 +83,11 @@ RUN echo " \
     " | tee /etc/apt/preferences.d/docker-ce
 
 # Install pinned version
-RUN  apt-get update --quiet && apt-get install --yes --no-install-recommends docker-ce docker-ce-cli docker-buildx-plugin
+# hadolint: Do not pin packages as they are requirements only for docker installation (and not of the rest of the ATH) and the effort required to track each version would be too huge
+# hadolint ignore=DL3008
+RUN apt-get update --quiet && apt-get install --yes --no-install-recommends docker-ce docker-ce-cli docker-buildx-plugin \
+    && apt-get clean \
+    && rm -rf /var/cache/apt
```

Comment:
The Docker CE version is pinned by the instruction above, which is a good thing (Docker actually have breaking changes some time to time: 22.x to 23.x did break a lot of things in a good way, but broke nevertheless).

But the "pinning" on APT is useful in a "long running instance" such as a VM. We can improve the current pinning by using pure-Docker form to make it easier to read and maintain.

My proposal is a refinment of the existing code and include removing the whole instruction above (`RUN echo "..." | tee /etc/apt/preferences.d/docker-ce` because I can't make a suggestion on unchanged code):

```suggestion
# Ensure that version is fixed
ARG DOCKER_VERSION=23.0.1
ARG DOCKER_BUILDX_VERSION=0.10.2
RUN apt-get update --quiet \
  && apt-get install --yes --no-install-recommends \
    docker-ce="5:${DOCKER_VERSION}*" \
    docker-ce-cli="5:${DOCKER_VERSION}*" \
    docker-buildx-plugin="${DOCKER_BUILDX_VERSION}*" \
  && apt-get clean \
  && rm -rf /var/cache/apt
```

=> no need to ignore package pinning (in this particular case, Docker-CE and Docker BuildX, pinning is important for the ATH usage otherwise it will break in 3 month with the next Docker CE update) in hadolint


WDYT?

Reply:
Sounds fine, I wasn't aware of that many breaking changes here 👀

Reply:
no problem, it was a good opportunity to point it out so the time you spent on fixing hadolint would be beneficial for you and the ATH